cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -117,6 +132,7 @@ func (o *indexArgs) BuildOptions() *build.Options {
 				"archived": marshalBool(o.Archived),
 				// Calculate repo rank based on the latest commit date.
 				"latestCommitDate": "1",
+				"tenantid":         strconv.Itoa(tenantID),
```

Comment:
very very minor, to match the other name styles this should be `tenantID`

Reply:
I am matching styles with repoid 7 lines up ;-)